build.gradle

```diff
@@ -69,6 +67,14 @@ plugins {
     id 'jacoco'
 }
 
+// import versions defined in https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/OpenSearchJavaPlugin.java#L94
+apply plugin: 'opensearch.java'
+// versions https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties
+ext {
+    jackson_version = versions.jackson
```

Comment:
Shouldn’t we conform to the same version name convention ?

Reply:
I did not get it. do you mean using version.jackson directly?

Reply:
yes 👍